docker/thirdparties/run-thirdparties-docker.sh

```diff
@@ -117,7 +117,10 @@ echo "Components are: ${COMPONENTS}"
 echo "Container UID: ${CONTAINER_UID}"
 echo "Stop: ${STOP}"
 
-COMPONENTS_ARR=(${COMPONENTS//,/ })
+OLD_IFS="${IFS}"
+IFS=','
+read -r -a COMPONENTS_ARR <<<"${COMPONENTS}"
+IFS="${OLD_IFS}"
```

Comment:
```suggestion
IFS=',' read -r -a COMPONENTS_ARR <<<"${COMPONENTS}"
```

Reply:
OK, I will modify it in next PR to save some regression test resource